modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue

```diff
@@ -108,11 +78,9 @@
 import AposInputMixin from 'Modules/@apostrophecms/schema/mixins/AposInputMixin.js';
 import cuid from 'cuid';
 import { klona } from 'klona';
-import draggable from 'vuedraggable';
```

Comment:
Draggability is still a needed feature we can't remove from inline arrays, although it will be disabled with `draggable: false` for these particular fields because it does not make sense for advanced permissions lists (the order is not significant).